x-pack/plugin/security/src/main/java/module-info.java

```diff
@@ -80,6 +80,7 @@
     exports org.elasticsearch.xpack.security.transport.extension to org.elasticsearch.internal.security;
     exports org.elasticsearch.xpack.security.transport to org.elasticsearch.internal.security;
     exports org.elasticsearch.xpack.security.audit to org.elasticsearch.internal.security;
+    exports org.elasticsearch.xpack.security.metric;
```

Comment:
Let's limit it only to `org.elasticsearch.internal.security`
```suggestion
    exports org.elasticsearch.xpack.security.metric to org.elasticsearch.internal.security;
```